pinot-common/src/main/java/com/linkedin/pinot/common/utils/SegmentName.java

```diff
@@ -21,6 +21,15 @@
   public static final String REALTIME_SUFFIX = "_REALTIME";
   public static final int REALTIME_SUFFIX_LENGTH = REALTIME_SUFFIX.length();
 
+  public static String buildBasic(String tableName, Object minTimeValue, Object maxTimeValue,
```

Comment:
Can we add a comment that this is for offline segments? I was confused for a bit as to why there was only one underscore when we join the segment name parts. :)